server/tests/page.rs

```diff
@@ -60,3 +60,35 @@ mod create_mutation {
         .await
     }
 }
+
+#[cfg(test)]
+mod pages_query {
+    use test_utils::*;
+
+    #[actix_rt::test]
+    async fn fetches_all_pages() {
```

Comment:
I'm just wondering if we would like to keep the pattern of other tests, in which the whole response is tested.
See for example [tests/subject.rs](https://github.com/serlo/serlo.org-database-layer/blame/main/server/tests/subject.rs) (don't even look at navigation.rs 😆 )